services/graylog/scripts/configure.py

```diff
@@ -139,17 +145,96 @@ def configure_email_notifications(_session, _headers):
     _mail_notification_id = [
         noti
         for noti in _r.json()["notifications"]
-        if noti["title"] == "Graylog " + env.str("MACHINE_FQDN") + " mail notification"
+        if noti["title"] == "Graylog " + MACHINE_FQDN + " mail notification"
     ][0]["id"]
 
     return _mail_notification_id
 
 
+def configure_slack_notification_channel(_session, _hed) -> str:
+    # Configure sending Slack notifications
+    if GRAYLOG_SLACK_WEBHOOK_URL != "":
+        assert GRAYLOG_SLACK_WEBHOOK_CHANNEL
+        assert GRAYLOG_SLACK_WEBHOOK_ICON_URL
+        print(
```

Comment:
TIP: you should always use `logging` instead of `print` to log information.

Reply:
you are right, but this specific enhancement will have to come another time, please feel free to re-raise this ;)